metasploit_data_models.gemspec

```diff
@@ -44,21 +43,18 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'pry'
 
 
-  s.add_runtime_dependency 'activerecord', '~>4.2.6'
-  s.add_runtime_dependency 'activesupport', '~>4.2.6'
+  s.add_runtime_dependency 'activerecord', '~>5.2.2'
+  s.add_runtime_dependency 'activesupport', '~>5.2.2'
   s.add_runtime_dependency 'metasploit-concern'
   s.add_runtime_dependency 'metasploit-model'
-  s.add_runtime_dependency 'railties', '~>4.2.6'
+  s.add_runtime_dependency 'railties', '~>5.2.2'
 
   # os fingerprinting
   s.add_runtime_dependency 'recog', '~> 2.0'
 
   # arel-helpers: Useful tools to help construct database queries with ActiveRecord and Arel.
   s.add_runtime_dependency 'arel-helpers'
 
-  # Fixes a problem with arel not being able to visit IPAddr nodes
-  s.add_runtime_dependency 'postgres_ext'
-
```

Comment:
@acammack-r7 please take a look at this change an validate that it can really be removed.

Reply:
Looks fine to me with it removed:
```
irb(main):005:0> Mdm::Host.first
  Mdm::Host Load (0.5ms)  SELECT  "hosts".* FROM "hosts" ORDER BY "hosts"."id" ASC LIMIT $1  [["LIMIT", 1]]
=> #<Mdm::Host id: 1, created_at: "2019-12-04 19:28:42", address: #<IPAddr: IPv4:10.20.36.98/255.255.255.255>, mac: nil, comm: "", name: nil, state: "alive", os_name: nil, os_flavor: nil, os_sp: nil, os_lang: nil, arch: nil, workspace_id: 1, updated_at: "2019-12-04 19:28:42", purpose: nil, info: nil, comments: nil, scope: nil, virtual_host: nil, note_count: 0, vuln_count: 1, service_count: 1, host_detail_count: 0, exploit_attempt_count: 1, cred_count: 0, detected_arch: nil, os_family: nil>
```